GraphQL-TSC.md

```diff
@@ -0,0 +1,253 @@
+* Approving new projects and working groups.
+* Creating, transferring, organizing, and renaming GitHub repos under the [GraphQL org](https://github.com/graphql).
+* Resolving technical or community issues that span multiple projects, or are deadlocked within a project.
+* Delegating decisions on publishing releases.
+
+The scope of the TSC should not include decisions that can be made within one of the projects, unless there is a specific issue that cannot be resolved and requires mediation. In general, the TSC prefers that decisions be made at the lowest possible level.
+
+#### Decision-making authority
+
+The following table is a rough outline of who is responsible for decisions. In keeping with the principle that decisions should be made at the lowest possible level and as efficiently as possible, unless there is a specific escalation to the TSC.
+
+| Decision | Who decides | Can it be delegated? |
+|---|---|---|
+| Approve a GraphQL Specification release | TSC | No |
+| Approve a sub-specification release | TSC or working group | Yes |
+| Release a reference implementation | TSC or maintainers | Yes |
+| Adding a new repo in the GraphQL organization | TSC | No |
+| Changes to this document | TSC | No |
+
+
+
+## Collaboration Tools
+
+### Mailing List
+
+The GraphQL TSC can be reached at [tsc@lists.foundation.graphql.org](https://lists.foundation.graphql.org/g/tsc). The mailing list is open to anyone who has signed the GraphQL Specification membership agreement (for [individual](https://individual-spec-membership.graphql.org/) or [corporate](https://corporate-spec-membership.graphql.org/) members), but anybody can view the archives.
+
+### Slack
+
+GraphQL maintains a [Slack](https://slack.graphql.org) for communication and collaboration, which is open for anyone to join. Once you join [Slack](https://slack.graphql.org), you can participate in any public channels.
+
+### Calendars
+
+GraphQL maintains a [public calendar](https://calendar.graphql.org) for TSC meetings. These meetings are open for anyone to join who has completed the GraphQL Specification Membership. To sign the agreement, simply open a pull request against the meeting agenda and add your name. The EasyCLA bot will direct you to the signature tool.
+
+Because we work in a highly distributed environment and will rarely meet in person, participants are encouraged to use video as appropriate.
+
+### Zoom
+
+GraphQL has a Zoom account which can be used by projects and working groups. Projects and working groups are encouraged to meet regularly and record their meetings for posting on the [GraphQL YouTube channel](https://youtube.graphql.org). To avoid collisions, please request that your meeting be added to the [GraphQL Calendar](https://calendar.graphql.org) by emailing [operations@graphl.org](mailto:operations@graphql.org). If two meetings are scheduled at the same time, the meeting on the calendar takes precedence.
```

Comment:
```suggestion
GraphQL has a Zoom account which can be used by projects and working groups. Projects and working groups are encouraged to meet regularly and record their meetings for posting on the [GraphQL YouTube channel](https://youtube.graphql.org). To avoid collisions, please request that your meeting be added to the [GraphQL Calendar](https://calendar.graphql.org) by emailing [operations@graphql.org](mailto:operations@graphql.org). If two meetings are scheduled at the same time, the meeting on the calendar takes precedence.
```